ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/WinMSSQL.py

```diff
@@ -84,8 +85,9 @@ def __init__(self, conn_info):
             | Get-ClusterParameter -Name VirtualServerName,InstanceName
             | Group ClusterObject | Select
             @{Name='SQLInstance';Expression={($_.Group | select -expandproperty Value) -join '\\'}},
-            @{Name='OwnerNode';Expression={($ownernode, $domain) -join '.'}}};
-        $cluster_instances | % {write-host \"instances:\"($_).OwnerNode\($_).SQLInstance};
+            @{Name='OwnerNode';Expression={($ownernode, $domain) -join '.'}},
+            @{Name='IPv4';Expression={(Test-Connection $ownernode -count 1).IPV4Address.ipaddresstostring}}};
```

Comment:
Maybe just do some testing there to see if the IPv4 field is set, and to use IPv6 if it isn’t? and also account for the possibility that neither is set/available?